### PR TITLE
Book a property routing

### PR DESCRIPTION
**Pull request by leatheronwillow**
Messy merges. Have put this

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 from lib.property import Property
 from lib.user_repository import UserRepository
 from lib.user import User
+from lib.booking_repository import BookingRepository
+from lib.booking_repository import BookingRepository
```
Comment (deesii) [inline app.py]: Duplication here.

Comment (deesii) [reply app.py]: Now pushed with deletion.

**Comment by deesii** [comment]
Can see that there has been routing added for listing the property. HTML for the bookings.html needs updating, but I will take this on, and update the tests accordingly.

Have made the deletion, with merges to main locally now pushed to origin.

File: app.py (hunk 1 above)
Comment (leatheronwillow) [reply app.py]: Thank you!

**Pull request merged**